tour_site_scraper.rb

```diff
@@ -1,92 +1,142 @@
+  all_tickets_details_lists = []
+  ticket_airlines&.each do |ticket_airline|
+    temp_ticket_airline_info = {}
+    number_of_ticket_found = 0
+
+    ticket_company_name = ticket_airline.find_element(:css, '.airline-name').text
+    number_of_ticket_found = ticket_airline.find_element(:css, '.toggle-btn-company').text.delete('^0-9').to_i
+    total_ticket_found += number_of_ticket_found
+    ticket_minimum_price = ticket_airline.find_element(:css, '.hdg-sup-price > b').text
+
+    temp_ticket_airline_info[:ticket_company_name] = ticket_company_name
+    temp_ticket_airline_info[:ticket_minimum_price] = ticket_minimum_price
+    temp_ticket_airline_info[:number_of_ticket_found] = number_of_ticket_found
+
+    ticket_flight_lists = []
+    ticket_airline_flights_lists = ticket_airline.find_elements(:css, '.Act_flight_list')
+    ticket_airline_flights_lists&.each do |ticket_flight|
+      flight_data = {}
+      flight_data['flight_code'] = ticket_flight.find_elements(:css, '.ticket-summary-row > span')[1].attribute("innerHTML")
+      flight_data['flight_price'] = ticket_flight.find_elements(:css, '.ticket-detail-item .ticket-detail-item-inner .ticket-price > label > b')[0].attribute("innerHTML")
+      flight_data['flight_seat'] = ticket_flight.find_elements(:css, '.ticket-detail-item .ticket-detail-item-inner .ticket-detail-type .ticket-detail-icon .icon-seat')[0].attribute("innerHTML")
+      flight_data['flight_changable_status'] = ticket_flight.find_elements(:css, '.ticket-detail-item .ticket-detail-item-inner .ticket-detail-type .ticket-detail-icon .icon-date')[0].attribute("innerHTML")
+      flight_data['flight_type'] = ticket_flight.find_elements(:css, '.ticket-detail-item .ticket-detail-item-inner .ticket-detail-type .ticket-detail-type-text .ticket-detail-type-text-ellipsis')[0].attribute("innerHTML")
+      ticket_flight_lists.push(flight_data)
+    end
+    temp_ticket_airline_info[:ticket_flight_lists] = ticket_flight_lists
+    all_tickets_details_lists.push(temp_ticket_airline_info)
   end
+  return all_tickets_details_lists, total_ticket_found
+end
 
-  # Scrap Returning Tickets Elements
-  ticket_summary_in = driver.find_elements(:css, '#Act_response_in .airline-name')
-  ticket_available_lists_in = driver.find_elements(:css, '#Act_response_in .toggle-btn-company')
+def save_scrap_data(tickets_out_lists, tickets_in_lists, departure_date, return_date)
+  all_ticket_out_lists = tickets_out_lists[0]
+  all_ticket_in_lists = tickets_in_lists[0]
+  total_ticket_out_found = tickets_out_lists[1]
+  total_ticket_in_found = tickets_in_lists[1]
+  puts  "Total tickets found for out is = " + total_ticket_out_found.to_s
+  puts  "Total tickets found for in is = " + total_ticket_in_found.to_s
+
+  DB.execute("INSERT INTO tickets_summary values(?, ?, ?, ?, ?, ?, ?, ? )", [nil, departure_date.to_s, return_date.to_s, TIME_FROM_OUT, TIME_TO_OUT, Time.now.strftime("%Y-%m-%d %H:%M:%S"), total_ticket_out_found, total_ticket_in_found])
```

Comment:
Please limit to 80 character per line for all. Ref: [Ruby Style Guide](https://github.com/rubocop/ruby-style-guide#maximum-line-length)